silverback/_cli.py

```diff
@@ -280,6 +280,86 @@ def cluster_health(cluster: ClusterClient):
     click.echo(yaml.safe_dump(cluster.health.model_dump()))
 
 
+@cluster.group(cls=SectionedHelpGroup)
+def docker():
+    """Manage Docker configuration"""
```

Comment:
I was thinking about this before, I think that `silverback cluster images` might be better subcommand group, because `image` is the configuration value you need to set for the bots, so it will be more obvious what that subcommand is for
```suggestion
def images():
    """Manage Docker image configuration"""
```

I could also see `silverback cluster registry` as an alternative

---

Not a huge fan of `docker` because it seems like it is just wrapping the `docker` command at first glance

Reply:
Also, I feel like this would be a good subcommand to add `images list`, `images info`, etc. eventually underneath (I know cluster doesn't currently support, but leave the space for this), so you can see if the cluster can fetch a particular image first before you set it in a bot config (and then watch it fail when deploying)

Reply:
I'm cool with dropping the docker branding.

Reply:
[c641cde](https://github.com/ApeWorX/silverback/pull/120/commits/c641cde0839535538ab0e3a82c5bca204480163b)